src/utils/lollipop.ts

```diff
@@ -50,6 +50,13 @@ export const checkIfLollipopIsEnabled = (
     TE.fromNullable(
       Error(`Cannot find configuration for service ${serviceId}`)
     ),
+    eventLog.taskEither.info((config) => [
+      `Check if lollipop is enabled`,
```

Comment:
For this name is better to use something that is searchable, something like: `lollipop.status.info`. 
So we can search it with queries.